fighthealthinsurance/utils.py

```
@@ -1180,56 +1213,57 @@ async def best_within_timelimit(
-    # Did we find a non empty best result?!?
+    def _score_done(done_tasks: Set[asyncio.Task[T]]) -> None:
+        nonlocal best_result_option, best_score
+        for task in done_tasks:
+            try:
+                result = task.result()
+                original_task = original_to_task[task]
+                score = score_fn(result, original_task)
+                if score > best_score:
+                    best_score = score
+                    best_result_option = result
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)